### DISCUSSION TITLE
Not Able to Upload 6 GB File using Git LFS Bash

### DISCUSSION THREAD

**Discussion by kumarbishal** [community/community#64938] [GitHub Education]
### Hi everyone,

Hi Team , Please help me . I am trying to upload 6GB Moodle_Data using Git LFS but It's not uploading even Branch is also not creating . Thanks in Advance

**Comment by arafajs** [comment]
First, let's make sure that Git LFS is installed and configured correctly. You can do this by running the following commands in your terminal:

```
git lfs install
git lfs config --global lfs.url=https://git-lfs.github.com
```
Once Git LFS is installed and configured, you can try uploading the file again. To do this, run the following command:

`git lfs track Moodle_Data`
This will tell Git LFS to track the Moodle_Data file and upload it to the remote repository.

If you are still having trouble uploading the file, you can try the following:

- Increase the maximum file size that can be uploaded to your repository. You can do this by editing the lfs.lfs.max_file_size setting in your .gitconfig file.
- Use a different branch to upload the file. Sometimes, problems with file uploads can be caused by conflicts with other files in the same branch.
- Contact GitHub support for help. If you have tried all of the above and you are still having trouble, you can contact GitHub support for help. They will be able to investigate the issue and help you resolve it.

- Git LFS documentation: https://git-lfs.github.com/
